components/nexus-security/src/main/java/org/sonatype/security/web/filter/authc/LogoutAuthenticationFilter.java

```diff
@@ -48,10 +53,17 @@ protected boolean onAccessDenied(ServletRequest request, ServletResponse respons
   }
 
   /**
-   * On postHandle, if we have subject, log it out.
+   * Ensures a logout request by always:
```

Comment:
Remove the word 'by' and change 'invalidating' to 'invalidates' on the first list item